index.js

```diff
@@ -67,6 +67,7 @@ var createLogger = require('./src/logger.js');
  *  Reporting
  */
 
+// TODO: Update this documentation
```

Comment:
I just put the TODO there because there is a separate item for updating jsdoc documentation.